app/components/version_milestones_component.rb

```diff
@@ -1,11 +1,28 @@
 # frozen_string_literal: true
 
 class VersionMilestonesComponent < ViewComponent::Base
-  def initialize(version:, title:, steps:)
+  def initialize(version:, milestones_presenter:)
     @version = version
```

Comment:
Is this instance variable no longer used? Seems like we use the version already set on the presenter currently, if I'm reading this correctly.

Reply:
Oh, nevermind. I see it's an attr_reader below!

Reply:
Version is the system version. User version is from the presenter. Sigh.